cookbooks/fastscape_eroding_box.prm

```diff
@@ -0,0 +1,172 @@
+# At the top, we define the number of space dimensions we would like to
```

Comment:
We changed the structure of our cookbooks a bit. Could you create a new directory `cookbooks/fastscape_eroding_box` and put the .prm file inside? Also can you either: 1. Link to a paper where we can read more about this model, or 2. Add a subdirectory `doc` inside the cookbook folder and write a paragraph (or more) about what is happening inside this model?

Reply:
This prm file is only a quick test of an eroding central block to make sure the coupling is working after installation or when making changes to the plugin. Is there a better place to put it?

Reply:
That sounds like a test. However to make this into an official test we would need to integrate downloading and building fastscape into the test suite. Can you just put it into its own subdirectory at least? Maybe add what you just wrote into either a README.md, or a very small file in cookbooks/fastscape_eroding_box/docs.